files/fr/web/css/color-scheme/index.html

```diff
@@ -0,0 +1,72 @@
+---
+title: color-scheme
+slug: Web/CSS/color-scheme
+browser-compat: css.properties.color-scheme
+translation_of: Web/CSS/color-scheme
+---
+<div>{{CSSRef}}</div>
+
+<p>La propriété CSS <strong><code>color-scheme</code></strong> permet d'indiquer les jeux de couleurs dans lesquel un élément peut être rendu sans risque.</p>
+<p>Les jeux de couleurs communément utilisés par les systèmes d'exploitation sont "light" (clair) et "dark" (sombre), ou "day mode" (mode jour) et "night mode" (mode nuit). Lorsqu'un utilisateur sélectionne un de ces jeux de couleurs, le système d'exploitation fait des ajustements à l'interface utilisateur. Cela comprend les contrôles de formulaires, les barres de défilement et les valeurs utilisées par le système de couleurs CSS.</p>
```

Comment:
(not blocking/mandatory: I'd use `<i>`s to wrap English terms.)

Reply:
LGTM

Reply:
fixed in https://github.com/mdn/translated-content/pull/1708/commits/cc2a98d61a693a04fa09a0d4bbfff1b6b2d36346